src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CreateCustomerComponent } from './create-customer';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { CustomerState, CreateCustomer } from 'src/app/modules/customer-management/store';
+import { FormBuilder } from '@angular/forms';
+import { Customer } from 'src/app/modules/shared/models/customer.model';
```

Comment:
The customer.model must be added in the index file in **'src/app/modules/shared/models/index.ts'**

Reply:
done